packages/vats/src/distributeFees.js

```diff
@@ -60,12 +63,21 @@ export function buildDistributor(treasury, bank, epochTimer, timer, params) {
     if (!queuedPayments.length) {
       return;
     }
-    E(bank).depositMultiple(
-      queuedAccounts.splice(0, depositsPerUpdate),
-      queuedPayments.splice(0, depositsPerUpdate),
-    );
 
-    scheduleDeposits();
+    const accounts = queuedAccounts.splice(0, depositsPerUpdate);
+    const payments = queuedPayments.splice(0, depositsPerUpdate);
```

Comment:
For later: Is splice the recommended way to pull multiple items off a queue in JS? Seems kind of low-level.

Reply:
The answer is embedded: "JS... Seems kind of low-level." :)  I believe that `splice` is the intended tool